content/en/monitors/guide/monitoring-available-disk-space.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Monitoring Available Disk Space
+kind: documentation
+---
+
+
+A common system metric to monitor is the available disk space on a given system or host. This guide will help you create a monitor alerting you when free disk space for a host falls below 10% for any host reporting to Datadog.
+
+To create the monitor for available disk space:
+
+1. In the navigation menu, click **Monitors**.
+2. Click **New Monitor**.
+3. Select **Metric** as the monitor type.
+4. Define the metric:
+     1. In the **Define the metric** section, input `system.disk.free` and avg by `host` (Query a).
+     2. Click **Add Query** and input `system.disk.total` and avg by `host` (Query b).
+     3. In the formula that appears, replace the default `a + b` with `a/b*100`.
+     
+         {{< img src="monitors/guide/monitoring_free_disk_space.png" alt="Query definition of system.disk.free and system.disk.total with formula a/b*100" style="width:80%;">}}
+    
+         Alernatively, you can use the **Source** tab and define the query as `avg:system.disk.free{*} by {host} / avg:system.disk.total{*} by {host} * 100`
```

Comment:
Is this an alternative to step 3 or an alternative to steps 1-3? Also, I think we can just delete this in favor of the other method.

Reply:
```suggestion
```